agnpy/spectra/spectra.py

```diff
@@ -473,3 +477,37 @@ def __str__(self):
             + f" - gamma_min: {self.gamma_min:.2e}\n"
             + f" - gamma_max: {self.gamma_max:.2e}"
         )
+
+
+class Interpolation(ElectronDistribution):
+
+    def __init__(
+        self,
+        gamma_data,
+        n_data
+    ):
+        self.gamma_data = gamma_data
+        self.n_data = n_data.value
```

Comment:
Maybe add a check that the distribution is provided in `cm-3`? Or convert to that value?